Assets/ProBuilder Data/Default Material Palette.asset.meta

```diff
@@ -1,5 +1,9 @@
 fileFormatVersion: 2
+<<<<<<< HEAD
+guid: ebd539e2ed5f2e84bb4a095437caa348
+=======
 guid: 862aba347cff0874293c26db96bb9c6a
+>>>>>>> master
 NativeFormatImporter:
   externalObjects: {}
   mainObjectFileID: 0
```

Comment:
Not sure what you are changing here, as you are changing stuff in the shared assets

Reply:
I have overwritten my .meta files with the .meta files from the master. Maro told me what to do.